src/components/App/AccountBase.js

```diff
@@ -0,0 +1,37 @@
+import React, { PureComponent } from 'react';
+import { AssignmentReturn, AssignmentReturned, SaveAlt } from '@material-ui/icons';
+import { Button, Avatar, Box } from '@material-ui/core';
+import { login, logout, saveUser } from '../../actions/AppStoreActions';
+
+class AccountBase extends PureComponent {
+    render() {
+        const user = this.props.user;
+        return (
```

Comment:
Moving forward, everything should be functional components.

```suggestion
const AccountBase = (props) => {
    const { users } = props;
    
    return (
```